pkg/openstack/actuator.go

```diff
@@ -37,10 +37,14 @@ import (
 	"github.com/openshift/cloud-credential-operator/pkg/operator/constants"
 	actuatoriface "github.com/openshift/cloud-credential-operator/pkg/operator/credentialsrequest/actuator"
 	"github.com/openshift/cloud-credential-operator/pkg/operator/utils"
+
+	yaml "gopkg.in/yaml.v2"
 )
 
 const (
 	rootOpenStackCredsSecretKey = "clouds.yaml"
+	rootOpenStackCloudName      = "openstack"
+	caCertFile                  = "/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem"
```

Comment:
I wondered about hard-coding this path here, and if other code pulls it from somewhere. However, a quick look in library-go reveals it's hard-coded in several places there too, so I'm assuming this is safe.

Reply:
Yeah, it is hardcoded in multiple places, including our installer, image registry and csi drivers...